flow-typed/box-ui-elements.js

```diff
@@ -980,3 +980,23 @@ type AdditionalVersionInfo = {
 };
 
 type OnVersionChange = (version: ?BoxItemVersion, additionalVersionInfo: ?AdditionalVersionInfo) => void;
+
+type Contact = {
```

Comment:
The name of this type is very generic but the data it represents is very specific to a single feature.